pages/api/cities.js

```diff
@@ -0,0 +1,49 @@
+const handler = (req, res) => {
+  const cityLocations = [
+    { city: 'Москва', branch: 92 },
```

Comment:
I'd better rename `city` to "name` + add `id` field. the `name` is easier to guess. Everything has a name